### ISSUE TITLE
Cannot view secret Key

### ISSUE THREAD

**Issue by MatthewJones0425** [seatgeek/api-support#76]
Trying to get up and running..     8-)

Client ID:   Shown in Full
Secret Key:   e9ca.......

Only 1st 4 characters show.  I cannot find an option to view the rest.   

Please advise.

Regards,

 MAJ

MERRY CHRISTMAS!!  HAPPY HOLIDAYS
HAPPY NEW YEAR!!

**Comment by dallasgutauckis** [comment]
You probably don't need your secret for anything, so it's probably not an issue. The client id is what's used on all requests. The secret key is not shown in full for security purposes, if for some reason you find that you need your secret key, please create a new application and copy the secret when it's displayed the first time.

**Comment by dallasgutauckis** [comment]
Also, happy new year to you too, Matthew. cc/ @zhammer

**Comment by MatthewJones0425** [comment]
I tried to create a new aPP it only shows first 4  - NEVER see the whole key.
An app I am using is asking for it.   Hmmm......  Let me try again and try without the key to see if it works.

**Comment by zackkitzmiller** [comment]
@MatthewJones0425 As Dallas said, you should never need your secret key, and I'd be highly suspicious of an app asking for yours. That said, when you generate a new key, your secret will be shown in the flash messaging as seen here. https://d.pr/i/qDKdia
